test/models/ui/comment_count_details_test.dart

```diff
@@ -0,0 +1,32 @@
+import "package:flutter_test/flutter_test.dart";
+import "package:proxima/models/ui/comment_count_details.dart";
+
+void main() {
+  group("Testing comment count details", () {
+    late CommentCountDetails comment;
+
+    setUp(() {
+      comment = const CommentCountDetails(count: 1324, isIconBlue: true);
```

Comment:
Do you think this should go in mock data? Or am I brainwashed by test refactors x)

Reply:
That's actually a good idea !
This will make the testing codebase cleaner and prevent potential future code duplication :+1: 

I have created a new test file `test/mocks/data/comment_count.dart` which contains a list with mock instances of `CommentCountDetails`.

I do that in f82cf5505b0d8d0dfd6c22ea702e9c2a3c115ff1.

Does this refactor seems ok to you ?

Reply:
awesome!